curriculum/challenges/english/16-the-odin-project/top-learn-data-types-and-conditionals/learn-data-types-and-conditionals-question-h.md

```diff
@@ -0,0 +1,57 @@
+---
+id: 65e9727a484dd50f720e6fed
+title: Learn Data Types and Conditionals Question H
+challengeType: 15
+dashedName: learn-data-types-and-conditionals-question-h
+---
+# --description--
+
+In any programming language, you need to compare values to make decisions. For exampe, if the weather is sunny,you will go out, otherwise you will stay at home. Another example would be to see if a user has enough points to level up in a game.
```

Comment:
```suggestion
In any programming language, you need to compare values to make decisions. For example, if the weather is sunny, you will go out, otherwise you will stay at home. Another example would be to see if a user has enough points to level up in a game.
```